server/app/assets/javascripts/questionBank.ts

```diff
@@ -139,11 +140,25 @@ class QuestionBankController {
     )
     questions.forEach((question) => {
       const questionElement = question as HTMLElement
-      const questionContents = questionElement.innerText
+
+      let questionFilterText
```

Comment:
**thought:** Not sure if this would make it cleaner, but you can also do something like:

```
const questionFilterText = questionElement.hasAttribute(
          QuestionBankController.RELEVANT_FILTER_TEXT_DATA_ATTR,
        ) ? questionElement.getAttribute(
          QuestionBankController.RELEVANT_FILTER_TEXT_DATA_ATTR,
        ) : questionElement.innerText
```

Reply:
Resolved using the `??` operator.